File: Server/CTSE-Exam-In-App/DTOs/Login/LoginResponse.cs
using CTSE_Exam_In_App.DTOs.Base;
using CTSE_Exam_In_App.Models;

namespace CTSE_Exam_In_App.DTOs.Login
{
    public class LoginResponse : BaseResponse
    {
        public User User { get; set; }
    }
}

File: Client/Android/app/src/main/java/com/example/examinapp/dataaccess/dtos/login/LoginResponse.java
package com.example.examinapp.dataaccess.dtos.login;

import com.example.examinapp.dataaccess.dtos.base.BaseResponse;
import com.example.examinapp.models.UserModel;

import java.util.Date;

public class LoginResponse extends BaseResponse {
    public UserModel user;

    public LoginResponse(boolean isSuccess, int status, String message, Date respondDateTime, UserModel user) {
        super(isSuccess, status, message, respondDateTime);
        this.user = user;
    }

    public UserModel getUser() {
        return user;
    }

    public void setUser(UserModel user) {
        this.user = user;
    }
}
